### PR TITLE
Discovery 8.0.0 (RPM)

### PR DESCRIPTION
**Pull request by lzap**
* [x] Nightly
* [x] 1.14

Deface dep is gone, once again.

```
[lzap@lzapx foreman_discovery]$ gem compare -k -b foreman_discovery 7.0.1 8.0.0
Fetching: foreman_discovery-7.0.1.gem (100%)
Compared versions: ["7.0.1", "8.0.0"]
  DIFFERENT files:
    7.0.1->8.0.0:
      * Deleted:
            app/overrides/
      * Added:
            db/migrate/20160805104605_rename_lock_template_setting.rb
              (!) Unexpected permissions: 100664
            db/migrate/20160925213030_change_discovery_widget_names.rb
              (!) Unexpected permissions: 100664
            db/migrate/20160927164411_define_explicit_length_for_discovery_rule_attributes.rb
              (!) Unexpected permissions: 100664
            db/migrate/20161006094714_add_constraints_on_subnets_smart_proxies.rb
              (!) Unexpected permissions: 100664
  DIFFERENT runtime dependencies:
    7.0.1->8.0.0:
      * Deleted:
            deface ["< 2.0"] (runtime)
```

### CODE REVIEW

File: rubygem-foreman_discovery/rubygem-foreman_discovery.spec [hunk 1]
```diff
@@ -34,14 +34,12 @@ URL:        http://github.com/theforeman/foreman_discovery
 Source0:    http://rubygems.org/downloads/%{gem_name}-%{version}%{?prever}.gem
 
 Requires:   foreman >= 1.13.0
```
Comment (domcleal) [inline rubygem-foreman_discovery/rubygem-foreman_discovery.spec]: 1.14.0, per engine.rb.

File: rubygem-foreman_discovery/rubygem-foreman_discovery.spec [hunk 2]
```diff
@@ -34,14 +34,12 @@ URL:        http://github.com/theforeman/foreman_discovery
...
-Requires:   %{?scl_prefix}rubygem(deface) < 2.0
 
 Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}rubygems
 
 BuildRequires: %{?scl_prefix_ruby}ruby(release)
 BuildRequires: foreman-plugin >= 1.13.0
```
Comment (domcleal) [inline rubygem-foreman_discovery/rubygem-foreman_discovery.spec]: 1.14.0

**Comment by lzap** [comment]
Thanks for quick review, @domcleal ! Trying to catch up with RCs...

**Comment by domcleal** [comment]
Merged and building, thanks @lzap.

**Pull request merged**